src/proposals/mips/mip-rewards-distribution/mipRewardsDistributionExample.json

```diff
@@ -0,0 +1,124 @@
+{
+    "1284": {
+        "addRewardInfo": {
```

Comment:
We are missing an `amount` field here so you know how much the `approve` call needs to be. It will be the amount of rewards per second multiplied by the total seconds in a reward epoch: 2,419,200

You could possibly do your own math, but in case reward epoch length ever changes maybe it's better for me to do the math for you in my tool.

Reply:
I'm sorry for some reason I must have missed the `amount` field in my review - you're right it's there.